cfn-resources/auditing/cmd/resource/resource.go

```diff
@@ -27,10 +28,10 @@ import (
 	log "github.com/mongodb/mongodbatlas-cloudformation-resources/util/logger"
 	progress_events "github.com/mongodb/mongodbatlas-cloudformation-resources/util/progressevent"
 	"github.com/mongodb/mongodbatlas-cloudformation-resources/util/validator"
-	mongodbatlas "go.mongodb.org/atlas/mongodbatlas"
+	atlasSDK "go.mongodb.org/atlas-sdk/v20230201008/admin"
 )
 
-var RequiredFields = []string{constants.ProjectID}
+var RequiredFields = []string{constants.ProjectID, constants.AuditFilter}
```

Comment:
not user about having a general constants package, sounds similar to having a util package that is not recommended in general

Reply:
not required anymore